cs3/app/registry/v1beta1/registry_api.proto

```diff
@@ -136,8 +136,8 @@ message ListSupportedMimeTypesResponse {
   // Opaque information.
   cs3.types.v1beta1.Opaque opaque = 2;
   // REQUIRED.
-  // The list of supported mime types with the apps which they can be opened with.
-  map<string, AppProviderList> mime_types = 3;
+  // The list of supported mime types and their properties.
+  MimeTypeList mime_types = 3;
```

Comment:
Instead of `MimeTypeList`, we can just use `repeated MimeTypeInfo` here. What do you think?

Reply:
It's equivalent indeed, but isn't it convenient to have a type for the list in Reva? or in that case we just have a `map`?

Reply:
Yep, we had to use this with a map because `map<string, repeated message>` isn't supported. It adds one more layer of abstraction which doesn't have much use. The access now would look like `response.MimeTypes.MimeTypes`, when we can have `response.MimeTypes`.